src/js/components/__tests__/APIErrorModal-test.js

```diff
@@ -0,0 +1,70 @@
+jest.dontMock('../APIErrorModal');
+jest.dontMock('../../events/AppDispatcher');
+jest.dontMock('../../constants/ActionTypes');
+jest.dontMock('../../stores/PreFlightStore');
+jest.dontMock('../../stores/DeployStore');
+jest.dontMock('../../stores/PostFlightStore');
```

Comment:
Can you remove these `dontMock`s that related to stores, since you disabled mocking of stores in `package.json`?